Source/AssetRipper.Processing/AnimatorControllers/VirtualAnimationFactory.cs

```diff
@@ -35,7 +35,7 @@ namespace AssetRipper.Processing.AnimatorControllers
 {
 	public static class VirtualAnimationFactory
 	{
-		private static Utf8String BlendTreeName { get; } = new Utf8String("BlendTree");
+		private static Utf8String BlendTreeName { get; } = new Utf8String("Blend Tree");
```

Comment:
Add this link: https://github.com/ds5678/Binoculars/blob/d6702ed3a1db39b1a2788956ff195b2590c3d08b/Unity/Assets/Models/binoculars_animator.controller#L106